lib/rules/require-parentheses-around-arrow-param.js

```diff
@@ -48,6 +48,9 @@ module.exports.prototype = {
     check: function(file, errors) {
         function isWrapped(node) {
             var openParensToken = file.getPrevToken(file.getFirstNodeToken(node));
+            if (node.type !== 'RestElement') {
+                openParensToken = file.getPrevToken(openParensToken);
```

Comment:
Having a really hard time visualizing/understanding how this works. Why do you need to do this for non-rest elements? If rest elements were failing, performing an action for non-rest elements doesn't seem right (though I see the tests passing). Any insight is appreciated.